mobify/platform-scaffold
# [Integration manager] login for merlins connector

This PR adds the login page actions to the merlins connector (form submission and fetch page content). 

 **JIRA**: https://mobify.atlassian.net/browse/WEB-1257
 **Linked PRs**: n/a

## Changes
- Moved login form submission action to the merlins connector
- Moved login page process action to the merlins connector

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- navigate to: https://www.merlinspotions.com/customer/account/login/
- the login and create account pages should load correctly and form submissions should work correctly
